ui/docs/people.md

```diff
@@ -0,0 +1,13 @@
4218?v=4" width="100px;" alt="alex"/><br /><sub><b>alex</b></sub></a><br /><a href="https://github.com/taskcluster/taskcluster/commits?author=alexandrasp" title="Code">💻</a></td><td align="center"><a href="https://alisha17.github.io/"><img src="https://avatars2.githubusercontent.com/u/13520250?v=4" width="100px;" alt="Alisha Aneja"/><br /><sub><b>Alisha Aneja</b></sub></a><br /><a href="https://github.com/taskcluster/taskcluster/commits?author=alisha17" title="Code">💻</a></td><td align="center"><a href="https://github.com/prachi1210"><img src="https://avatars3.githubusercontent.com/u/14016564?v=4" width="100px;" alt="Prachi Manchanda"/><br /><sub><b>Prachi Manchanda</b></sub></a><br /><a href="https://github.com/taskcluster/taskcluster/commits?author=prachi1210" title="Code">💻</a></td><td align="center"><a href="https://github.com/srfraser"><img src="https://avatars1.githubusercontent.com/u/5933384?v=4" width="100px;" alt="Simon Fraser"/><br /><sub><b>Simon Fraser</b></sub></a><br /><a href="https://github.com/taskcluster/taskcluster/commits?author=srfraser" title="Code">💻</a></td><td align="center"><a href="https://github.com/ydidwania"><img src="https://avatars1.githubusercontent.com/u/22861049?v=4" width="100px;" alt="Yashvardhan Didwania"/><br /><sub><b>Yashvardhan Didwania</b></sub></a><br /><a href="https://github.com/taskcluster/taskcluster/commits?author=ydidwania" title="Code">💻</a></td></tr><tr><td align="center"><a href="https://cynthiapereira.com"><img src="https://avatars3.githubusercontent.com/u/1923666?v=4" width="100px;" alt="Cynthia Pereira"/><br /><sub><b>Cynthia Pereira</b></sub></a><br /><a href="https://github.com/taskcluster/taskcluster/commits?author=cynthiapereira" title="Code">💻</a></td><td align="center"><a href="https://github.com/hashi93"><img src="https://avatars2.githubusercontent.com/u/12398942?v=4" width="100px;" alt="Hashini Galappaththi"/><br /><sub><b>Hashini Galappaththi</b></sub></a><br /><a href="https://github.com/taskcluster/taskcluster/commits?author=hashi93" title="Code">💻</a></td><td align="center"><a href="https://github.com/fiennyangeln"><img src="https://avatars1.githubusercontent.com/u/24544912?v=4" width="100px;" alt="Fienny Angelina"/><br /><sub><b>Fienny Angelina</b></sub></a><br /><a href="https://github.com/taskcluster/taskcluster/commits?author=fiennyangeln" title="Code">💻</a></td><td align="center"><a href="https://github.com/kanikasaini"><img src="https://avatars2.githubusercontent.com/u/20171105?v=4" width="100px;" alt="Kanika Saini"/><br /><sub><b>Kanika Saini</b></sub></a><br /><a href="https://github.com/taskcluster/taskcluster/commits?author=kanikasaini" title="Code">💻</a></td><td align="center"><a href="https://github.com/Biboswan"><img src="https://avatars2.githubusercontent.com/u/22202556?v=4" width="100px;" alt="Biboswan Roy"/><br /><sub><b>Biboswan Roy</b></sub></a><br /><a href="https://github.com/taskcluster/taskcluster/commits?author=Biboswan" title="Code">💻</a></td></tr></table>
```

Comment:
html tags are not picked up by mdx. Are we able to switch `<a />` to `[]()`. I think this would fix our issue.

Reply:
Not if we want to use allcontributors, no.  We could easily do so with `yarn generate` but then we lose the benefit of the bot.  (We could still use the bot, but would have to run `yarn generate` on every PR from the bot, which pretty much defeats the purpose).

Will mdx-style not find the `<a>` elements, either?  If so, the we may be on our own and not able to use allcontributors..

Reply:
(I can try it and find out, just wondering if you were suggesting it wouldn't work)

Reply:
> Is https://github.com/jxnblk/mdx-style the right way to accomplish this? I can't figure out any other approach..

It would probably work. I would probably just add a `<style />` tag https://developer.mozilla.org/en-US/docs/Web/HTML/Element/style instead.

Reply:
It doesn't appear that's valid JSX (or maybe MDX or MDHASH or HASH.. not sure):
```
SyntaxError: /home/dustin/p/taskcluster/ui/docs/people.md: Unexpected token, expected "}" (25:17)

  23 | <style>
  24 | a {
> 25 |   text-decoration: none;
     |                  ^
  26 | }
  27 | </style>
```

Reply:
And using dangerouslySetInnerHTML causs it to get stuck on the comment:

```
  25 |   text-decoration: none;
  26 | }`}} />
> 27 | <!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
     |  ^
  28 | <!-- prettier-ignore -->
```